UKHomeOffice/sgmr-service
# SPCR-205 Implement mobile menu

## Description
Implements the mobile menu as specified within the GDS template.

## To Test
- Sign in to the service
- Switch to a mobile screen (either through a device or in a desktop browser)
- You should see a menu button
- Click on the menu button
- Navigation options should appear
- Click on the options and and you should navigate through the page as you would on a desktop

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
